src/Events/Models/Subscription.cs

```diff
@@ -54,6 +54,11 @@ public class Subscription
         /// </summary>
         public DateTime Created { get; set; }
 
+        /// <summary>
```

Comment:
What are the criterias for ok? - That the endpoint has validated the PUT request?

Reply:
Yes, the endpoint replied with 200 OK.